### ISSUE TITLE
Updates to Entity values are not persisted

### ISSUE THREAD

**Issue by fatbatman** [metasoarous/catalysis#39]
After doing a git clone 
lein repl
(run)

The Todo list shows but updates to entities fail with;

![image](https://user-images.githubusercontent.com/797736/27806776-81c0a6bc-6034-11e7-9d02-6464878e0e0b.png)

**Comment by fatbatman** [comment]
Which seems to be due to the way the logging is configured?

![image](https://user-images.githubusercontent.com/797736/27806808-abfc717c-6034-11e7-85e5-3065961e5e39.png)

**Comment by metasoarous** [comment]
I don't think it has anything to do with the way logging is called. That's just the odd way exceptions bubble up in JS with try catch. I think... I could be wrong. Do you have any thoughts on this @bamarco?

**Comment by fatbatman** [comment]
Does anyone happen to remember the last time this working on any DatSys branch/release?
It was working in an old Cayalysis branch I'd prefer not to go that far back.

**Comment by fatbatman** [comment]
Somewhere in the call stack a nil gets added to the server-event giving it a length of 3 and so failing the validation.
I couldn't see where that was happening so it's possible it's some kind of clojurescript wierdness
Anyway, this fixes it https://github.com/fatbatman/datsync/commit/ed549dd89fe2d7039149112b7fb6c5123554b89c

**Comment by bamarco** [comment]
@metasoarous This seems like it might be somewhere in the datview code that is confused about sente events vs datsync events, maybe?

**Comment by metasoarous** [comment]
@fatbatman - good find! Thanks for tracking that down! The situation is that I needed to send transaction metadata at some point, and so I added a third entry to the vector. I had it working at _some_ point, but something else must have changed that made that break. I wasn't totally satisfied with the hack I baked in anyway though. It might be better to use a separate event name in that case, something like `[:send-tx-with-metadata [tx-forms metadata]]`.

@bamarco - It's possible that's why this started breaking.
